mqttclients/dotnet/MQTTnet.Client.Extensions/X509ChainValidator.cs

```diff
@@ -27,23 +27,19 @@ internal static bool ValidateChain(MqttClientCertificateValidationEventArgs cvAr
             if (cvArgs.SslPolicyErrors == SslPolicyErrors.RemoteCertificateChainErrors)
             {
                 bool chainValidated = false;
-
-                cvArgs.Chain.ChainPolicy.RevocationFlag = X509RevocationFlag.ExcludeRoot;
-                cvArgs.Chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
-                cvArgs.Chain.ChainPolicy.VerificationTime = DateTime.UtcNow;
+                cvArgs.Chain.Reset();
                 cvArgs.Chain.ChainPolicy.TrustMode = X509ChainTrustMode.CustomRootTrust;
+                cvArgs.Chain.ChainPolicy.VerificationFlags = X509VerificationFlags.IgnoreEndRevocationUnknown;
```

Comment:
Please add a Trace to show that revocation checks are now disabled.
Also, please add a comment explaining the dangers of using this flag if the CA certificate _does_ include revocation checks such as CRL or OCSP. It is recommended that, when using this flag, an external CA certificate revocation mechanism is used (e.g. `apt ca-certificates` or Windows Update).

__Important security__: If the CA certificate does include CRL/OCSP or other revocation mechanisms supported by the TLS stack _and_ the certificate _has been revoked_, this flag will ignore any such revocation.

Reply:
The flag is self explanatory, because it is not hidden from user, and user can very well check docs. The word 'Ignore' says it all. 
Here is what we could do, and I leave it up to Rido. 
1) consider adding a Trace, this seems like good practice and gives the user a chance to observe behavior at runtime, and inquire in case 
2) adding the comment is optional

Case closed.